cmd/vm.go

```diff
@@ -21,62 +24,29 @@ var vmCmd = &cobra.Command{
 	Long:  `VM commands for kvmservice`,
 }
 
-// vmAddCmd represents the vm command for vm onboarding
-var vmAddCmd = &cobra.Command{
-	Use:   "add",
-	Short: "add/onboard a new vm for nonk8s control plane",
-	Long:  `add/onboard a new vm for nonk8s control plane`,
-	RunE: func(cmd *cobra.Command, args []string) error {
-		return nil
-	},
-}
-
-// vmDelCmd represents the vm command for vm onboarding
-var vmDelCmd = &cobra.Command{
-	Use:   "delete",
-	Short: "delete/offboard a vm from nonk8s control plane",
-	Long:  `delete/offboard a vm from nonk8s control plane`,
-	RunE: func(cmd *cobra.Command, args []string) error {
-		return nil
-	},
-}
-
 // vmScriptCmd represents the vm command for script download
 var vmScriptCmd = &cobra.Command{
 	Use:   "getscript",
 	Short: "download vm installation script for nonk8s control plane",
 	Long:  `download vm installation script for nonk8s control plane`,
 	RunE: func(cmd *cobra.Command, args []string) error {
-		if err := vm.GetScript(client, scriptOptions); err != nil {
+		ip := HttpIP
+		isNonK8sEnv := IsNonK8sEnv
```

Comment:
There is no point in having a local variable `isNonK8sEnv`? The same change in above places.

Reply:
Addressed @nyrahul 
That was kept for vaildation by mistake.